frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts

```diff
@@ -33,9 +63,169 @@ describe('git import flow', () => {
   });
 
   it('public git normal flow', async () => {
+    await switchPerspective(Perspective.Developer);
+    expect(sideHeader.getText()).toContain('Developer');
+    await navigateImportFromGit();
+    await browser.wait(until.textToBePresentInElement(importFromGitHeader, 'Import from git'), waitForElement);
+    expect(importFromGitHeader.getText()).toContain('Import from git');
+    await enterGitRepoUrl(testData.gitRepoUrl);
+    debugger;
```

Comment:
nit: remove this

Reply:
Thanks for reminding me :-)  I will update it and will make a note of it for the next PR's as well.